liota/entities/registered_entity.py

```diff
@@ -36,6 +36,16 @@ def __init__(self, ref_entity, ref_dcc, reg_entity_id):
         self.ref_entity = ref_entity
         self.ref_dcc = ref_dcc
         self.reg_entity_id = reg_entity_id
+        self.parent = None
```

Comment:
Consider in the future, Metric could be the parent of another parent. parent will be a common thing for both RegisteredEntity and RegisteredMetric. It seems to me correct to put parent in the base class of RegistereEntity. 

I agree that in python we can directly in other module call object attribute even in base class. I'll do that way.